src/renderer/reducers/register.ts

```diff
@@ -1,21 +1,28 @@
-import {ISigningKeyMnemonicAction, ISigningKeyAction, IWithdrawalKeyAction} from "../actions";
+import {ISigningKeyMnemonicAction, ISigningKeyAction, 
+    IWithdrawalKeyAction, IFailedVerificationAction} from "../actions";
 import {RegisterActionTypes} from "../constants/action-types";
 import {Action} from "redux";
 
 export interface IRegisterState {
     mnemonic: string,
     signingKey: string,
     withdrawalKey: string
+    failedVerification: boolean
 }
 
 const initialState: IRegisterState = {
     mnemonic: "",
     signingKey: "",
-    withdrawalKey: ""
+    withdrawalKey: "",
+    failedVerification: false
 };
 
 export const registerReducer = (state = initialState, action: Action<RegisterActionTypes>): IRegisterState => {
     switch (action.type) {
+        case RegisterActionTypes.STORE_FAILED_VERIFICATION:
+            return Object.assign({}, state, {
+                failedVerification: (action as IFailedVerificationAction).payload.failedVerification
```

Comment:
you should probably put `mnemonic: ""`
and just put `failedSigningMnemonicVerification: true` - no need for payload